src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include <sstream>
 
-#ifdef HAS_FIND_MODE_API
+#ifdef MIGRAPHX_HAS_FIND_MODE_API
 extern "C" miopenStatus_t miopenHiddenSetConvolutionFindMode(miopenConvolutionDescriptor_t convDesc,
```

Comment:
You need to add `// NOLINT` to suppress the tidy warnings.

Reply:
Oops you are right, done in 1cf219e.